minimum_jerk_trajectory_planner/include/minimum_jerk_trajectory_planner/trajectory_planners.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include "pose.hpp"
+#include <math.h>
+#include <stdlib.h>
+#include <string.h>
+#include <vector>
+
+
+namespace minimum_jerk
+{
+    class TrajectoryPlanner
+    {
+    public:
+        std::vector<double> list_t;
```

Comment:
probably should be renamed `list_timestamps` (i think `list_t` is a built-in name), should also be private with a getter